tests/Doxyfile

```diff
@@ -656,6 +656,7 @@ WARN_LOGFILE           =
 INPUT                  = ../Bootloader/firmware/src \
                          ../common \
                          ../firmware/src \
+                         ../firmware/src/system_config/ethernet_sk2/system_settings.h \
```

Comment:
How about adding tests/system_config/system_settings.h or at least the number 8 one instead then, rather than the dev one, which is less likely to get updates to Doxygen?